api/src/integration/java/org/pmiops/workbench/FireCloudIntegrationTest.java

```diff
@@ -1,6 +1,7 @@
 package org.pmiops.workbench;
 
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
```

Comment:
In other tests, we use `org.junit.jupiter.api.Assertions.assertThrows;`

Reply:
Good catch!  I have updated all of the remaining `org.junit.Assert` references to be `org.junit.jupiter.api.Assertions`